front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -235,15 +246,19 @@ export default function AssistantBuilder({
   dustApps,
   initialBuilderState,
   agentConfigurationId,
+  flow,
 }: AssistantBuilderProps) {
   const router = useRouter();
   const sendNotification = React.useContext(SendNotificationsContext);
   const slackDataSource = dataSources.find(
     (ds) => ds.connectorProvider === "slack"
   );
+  const initialScope =
```

Comment:
This is a bit ill named. Because this is not the initialScope if the assistant already exists.

Maybe call it defaultScope?